iOS_SDK/OneSignalSDK/Source/OneSignalTracker.m

```diff
@@ -88,7 +77,7 @@ + (void)applicationBecameActive {
     if (OSSessionManager.sharedSessionManager.appEntryState != NOTIFICATION_CLICK)
         OSSessionManager.sharedSessionManager.appEntryState = APP_OPEN;
    
-    lastOpenedTime = [NSDate date].timeIntervalSince1970;
+    [OSSessionManager setLastOpenedTime:[[NSDate date] timeIntervalSince1970]];
```

Comment:
Calls to the session manager should be through the shared instance like the line above via `[OSSessionManager.sharedSessionManager setLastOpenedTime:[[NSDate date] timeIntervalSince1970]]`